appraisal.lic

```diff
@@ -25,6 +26,42 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
     true
   end
+  
+  def train_appraisal_with_pouches?
+    settings = get_settings
+    return false unless settings.train_appraisal_with_pouches
+    full_pouch_container = settings.full_pouch_container
+    if !full_pouch_container
+      echo "Must set full_pouch_container in YAML"
+      return false
+    end      
+    if full_pouch_container == settings.spare_gem_pouch_container
```

Comment:
Well I'd just really like to have them in validate. These are errors we can detect before the script runs, and frankly most people never notice error messages when scripts run and just say "It didn't work."